app/mixins/event-wizard.js

```diff
@@ -41,23 +42,20 @@ export default Mixin.create(MutableArray, CustomFormMixin, {
   },
 
   /**
-   * Save event & related data
-   *
+   * Save event & related data 
+   * 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5583663)